released/packages/coq-kruskal-almostfull/coq-kruskal-almostfull.1.0/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "2.0"
+name: "Coq-Kruskal-AlmostFull"
+synopsis: "Base Coq library for manipulating Almost Full relations"
+description: """
+   This library formalizes ground results about Almost Full relations (AF) in Coq 8.14+, up to Dickson's lemma.
+"""  
+maintainer: ["Dominique Larchey-Wendling (https://github.com/DmxLarchey)"] 
+authors: "Dominique Larchey-Wendling (https://github.com/DmxLarchey)"
+license: "MPL-2.0"
+homepage: "https://github.com/DmxLarchey/Kruskal-AlmostFull/"
+bug-reports: "https://github.com/DmxLarchey/Kruskal-AlmostFull/issues"
+dev-repo: "git+https://github.com:DmxLarchey/Kruskal-AlmostFull/"
+
+build: [
+  [ make "-j%{jobs}%" "type"]
```

Comment:
This seems to indicate that the `Type` version is built, but you still state `logpath:KruskalAfProp`. Doesn't building the `Type` version exclude the `Prop` version?

Reply:
Well the `make install` actually builds and installs both `logpath:KruskalAfProp` and `logpath:KruskalAfType`. The same code base is compiled except some file `base_implem.v` which contains notations to accomodate of the `Prop` vs `Type` slight differences.

Reply:
`make install` calls `make prop_install` and then `make type_install`.

Reply:
One __cannot run__ the build process for `Prop` (ie `make prop`) and for `Type` (ie `make type`) simultaneously because the `_CoqProject` file is the `base/base_implem.v` files are specific to each build.

The install process is sequential.